pom.xml

```diff
@@ -50,4 +50,13 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>maven-plugin</artifactId>
+      <version>[2.0,)</version>
```

Comment:
I'm not sure if all components of Jenkins infra support such expression correctly. CC @rtyler and @daniel-beck